classes/class-wc-connect-help-view.php

```diff
@@ -78,49 +78,28 @@ protected function get_health_items() {
 			}
 			$health_items['woocommerce'] = $health_item;
 
-			// Jetpack
-			// Only one of the following should present
-			// Check that Jetpack is active
-			// Check that Jetpack is connected
-			include_once ABSPATH . 'wp-admin/includes/plugin.php'; // required for is_plugin_active
-			$is_connected = WC_Connect_Jetpack::is_active() || WC_Connect_Jetpack::is_development_mode();
-			if ( ! is_plugin_active( 'jetpack/jetpack.php' ) ) {
-				$health_item = array(
-					'state'   => 'error',
-					'message' => sprintf(
-						__( 'Please install and activate the Jetpack plugin, version %s or higher', 'woocommerce-services' ),
-						WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION
-					),
-				);
-			} elseif ( version_compare( JETPACK__VERSION, WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION, '<' ) ) {
+			if ( WC_Connect_Jetpack::is_offline_mode() ) {
```

Comment:
Adding offline mode as a separate possible status because this mode can be enabled when a dev mode is on (using `JETPACK_DEV_DEBUG` or `WP_LOCAL_DEV`) but also when simply running the site on a local TLD like `.test` or `.local`.